src/data.rs

```diff
@@ -325,6 +335,7 @@ impl Data {
                 }
             }
         }
+        let _close_result = workbook.close();
```

Comment:
Use `?` op for error handling.

Change
```
let _close_result = workbook.close();
```
to
```
workbook.close()?;
```

Reply:
Done :)